person.rb

```diff
@@ -0,0 +1,29 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def can_use_services?
+    true if @age >= 18 || parent_permission == true
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] **[OPTIONAL]** I forgot to mention in the first review that you should not make this method private. I got an error saying that I'm trying to access the private method `correct_name` in the person class. So please kindly make this method `public` before your next milestone.